test/SdkTests/TestConfig.xml

```diff
@@ -141,9 +141,30 @@
             Skip="true"
             Issue=""
             Reason="Need all .NET core runtime"/>
-    <Method Name="Microsoft.NET.Publish.Tests.GivenThatWeWantToRunILLink.ILLink_verify_analysis_warnings_hello_world_app"
+
+    <Method Name="Microsoft.NET.Publish.Tests.GivenThatWeWantToPublishAFrameworkDependentApp.It_publishes_with_or_without_apphost"
```

Comment:
@marcpopMSFT @wtgodbe this is not needed for main right?

Reply:
Not needed in main

Reply:
@sfoslund could you revert that? then we can merge

Reply:
Sure, reverted.